src/main/kotlin/turku/forge/babyproject/resources/Resource.kt

```diff
@@ -0,0 +1,33 @@
+package turku.forge.babyproject.resources
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.springframework.hateoas.LinkRelation
+import org.springframework.hateoas.RepresentationModel
+
+/**
+ * This is a `RepresentationModel` abstraction.
+ *
+ * The point of these are to be JSON representations of the values
+ * we send to the api consumer. Any class that extends this will automatically
+ * be serialized to a `hal+json` media container.
+ *
+ * since we don't yet know what media container we will use
+ * this class serves as a abstraction for the media container we are currently using.
+ *
+ * Its likely that this will be removed once we have chosen a media container
+ * but we may keep it who knows
```

Comment:
This part of the comment could actually be removed. The statement here holds true for almost all code in any project: it might be removed, it might be kept; stating this is just so obvious that it doesn't add much value, at least not to me. :sweat_smile:

Reply:
Yea that is true. This entier file has been removed in my upp and comming commit. But I do agree it adds no benefit to state that.